bucket/goneovim.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v0.4.12",
```

Comment:
Version here usually does not have the word v.

Please fix that.

It should be 0.4.12 only